### PR TITLE
Small updates for Windows support

### PR DESCRIPTION
**Pull request by aiyengar2**
Comments out global.cattle.windows.enabled so that users can set it to false in Rancher Dashboard UI.

Adds dummy `os` values to support marking images as Windows images for private registry script.

Related Issues: https://github.com/rancher/rancher/issues/32274, https://github.com/rancher/rancher/issues/32265, https://github.com/rancher/rancher/issues/32280

### CODE REVIEW

File: packages/rancher-windows-exporter/charts/values.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ clients:
   image:
     repository: rancher/windows_exporter-package
     tag: v0.0.1
+    os: "windows"
```
Comment (nickgerace) [inline packages/rancher-windows-exporter/charts/values.yaml:24]: Is this a windows only image?

File: packages/rancher-wins-upgrader/generated-changes/patch/values.yaml.patch [hunk 2]
```diff
@@ -0,0 +1,10 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -20,6 +20,7 @@
+     # TODO(aiyengar2): replace with an image that just contains wins
+     repository: rancher/wins
+     tag: v0.1.0
++    os: "windows"
```
Comment (nickgerace) [inline packages/rancher-wins-upgrader/generated-changes/patch/values.yaml.patch:7]: Is this a windows only image?

**Review by nickgerace — APPROVED** [review]

File: packages/rancher-wins-upgrader/generated-changes/patch/values.yaml.patch (hunk 2 above)
Comment (aiyengar2) [reply packages/rancher-wins-upgrader/generated-changes/patch/values.yaml.patch:7]: yes

File: packages/rancher-windows-exporter/charts/values.yaml (hunk 1 above)
Comment (aiyengar2) [reply packages/rancher-windows-exporter/charts/values.yaml:24]: yes

**Comment by aiyengar2** [comment]
Closing in favor of https://github.com/rancher/charts/pull/1150, which adds 1 more commit.

**Pull request closed without merging**